src/core/dbus.c

```diff
@@ -1181,6 +1181,27 @@ int bus_verify_manage_units_async(Manager *m, sd_bus_message *call, sd_bus_error
                         error);
 }
 
+int bus_verify_manage_units_with_name_async(
```

Comment:
hmm, so we have pretty much this routing already:

bus_verify_manage_units_async_full(), and it's a bit more elaborate.

no idea why it is placed in dbus-util.c though, rather than dbus.c where the other calls are. THis seems to be weird since the call was added in 88ced61bf9673407f4b15bf51b1b408fd78c149d.

I guess this deserves some clean-up, i.e. move things closer together (and maybe give things a less confusing name, i.e. bus_verify_manage_units_async_full() should probably stop saying "units" in plural. maybe renamed to bus_unit_verify_manager() or so?

(i can understand if you don't want to clean this up, nd it's fine, if you just start making use of that function, and don't clean this up, but the cleaning up would be greatly welcome)

Reply:
I've moved bus_verify_manage_units_async_full to dbus.{c|h}, and changed its parameters. I think function name is better kept as is to match polkit action (`org.freedesktop.systemd1.manage-units`).

Reply:
Hmm, actually I think it lives in dbus-util since it uses `Unit`?